digits/dataset/tasks/create_db.py

```diff
@@ -53,9 +53,13 @@ def __init__(self, input_file, db_name, image_dims, **kwargs):
 
         self.entries_count = None
         self.distribution = None
+        self.create_db_log_file = "create_%s.txt" % db_name
```

Comment:
Yes, sorry I read the first comment too quickly. I used the .txt extension because in Firefox it shows the file automatically without asking the user to download/choose an external tool to open it. But you're right, it's better to keep consistency in the project.